ui/app/product-milestones/product-milestone.module.js

```diff
@@ -52,16 +53,48 @@
         resolve: {
           productMilestone: ['ProductMilestoneResource', '$stateParams', (ProductMilestoneResource, $stateParams) =>
             ProductMilestoneResource.get({ id: $stateParams.productMilestoneId }).$promise
-          ],
+          ]
+        }
+      })
+      .state('products.detail.product-versions.detail.milestone.detail.default', {
+        url: '/',
+        component: 'pncProductMilestoneDetailDetailsTab',
+        data: {
+          title: '{{ productMilestone.version }} | {{ product.name }} | Milestone',
+          displayName: 'Details',
+        },
+        resolve: {
+          closeResults: ['ProductMilestoneResource', '$stateParams', (ProductMilestoneResource, $stateParams) =>
```

Comment:
`closeResults` should be renamed to `latestCloseResult`

Reply:
basically, the response of the endpoint remains to be a list but contains only one data, thus not a specific latestCloseResult. if we rename it into latestCloseResult, it may cause some misunderstanding.